packages/block-editor/src/rating-scale-answer/edit-checkbox.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { FormCheckbox } from '@crowdsignal/blocks';
+import { useColorStyles } from '@crowdsignal/styles';
+
+const EditCheckboxAnswer = ( {
+	attributes,
+	className,
+	multipleChoice,
+	onChange,
+	onReplace,
+	onSplit,
+	onDelete,
+} ) => (
+	<FormCheckbox.Label
+		as={ 'div' }
+		className={ className }
+		style={ useColorStyles( attributes ) }
+	>
+		<FormCheckbox type={ multipleChoice ? 'checkbox' : 'radio' } />
```

Comment:
Hmm... I'm not sure why you'd want a multiple choice rating but feel free to enlighten me.

Reply:
Matrixes do have a "multiple answer" option. While this is not the case for ratings, I'd thought we could keep that here if we are to use this component in the future.

Reply:
Then again, the file is not used by any component, so I might just remove the whole thing